network_setup.sh

```diff
@@ -111,6 +111,8 @@ $MIRTE_SRC_DIR/mirte-install-scripts/usb_ethernet.sh
 if [ ! -f /etc/ssid ] || [[ $(cat /etc/hostname) == "Mirte-XXXXXX" ]]; then
 	UNIQUE_ID=$(openssl rand -hex 3)
 	MIRTE_SSID=Mirte-$(echo ${UNIQUE_ID^^})
+	# replace al 0s with As to make it clearer for users when looking at the leds, as 0 is difficult to recognize
+	MIRTE_SSID=${MIRTE_SSID//0/A}
```

Comment:
I would not replace 0's with A's, but leave out the 0's from the RNG. I do not think there is an easy way of doing this with openssl. So I would replace:

UNIQUE_ID=$(tr -cd "1-9A-F" < /dev/urandom | head -c 6)

to make sure 0's are not in there.

Reply:
I've changed it and I'm currently generating an SD to test that they differ